main/Card/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  ActivityIndicator,
+  ImageSourcePropType,
+  ImageStyle,
+  TextStyle,
+  ViewStyle,
+} from 'react-native';
+import React, { FC, ReactNode, ReactNodeArray } from 'react';
+
+import { FlattenSimpleInterpolation } from 'styled-components';
+import styled from 'styled-components/native';
+
+import styles from './styles';
+
+interface IContainerProps {
+  raised?: boolean;
+  outlined?: boolean;
+}
+
+const Container = styled.View<IContainerProps>`
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: #fff;
+  width: 256px;
+  ${({ outlined, raised }): FlattenSimpleInterpolation =>
+    outlined ? styles.border : raised ? styles.raisedShadow : styles.shadow}
```

Comment:
It causes warning as `Node of type rule not supported as an inline style`. 
Find another way.

Reply:
The cause of this problem was shadow-offset!
fixed it by 
1842cbf